Strings/leftTrim.c

```diff
@@ -0,0 +1,42 @@
+/*
+LEFT TRIM
+This program eliminates extra spaces from left side of the string.
+
+INPUT FORMAT : one string
+OUTPUT FORMAT : one string 
+SAMPLE INPUT :           left trim program. (size of string is 37 so input string lenght should be <=35)
+SAMPLE OUTPUT :(left trim program.)
+*/
+
+#include<stdio.h>
+#include<string.h>
+void leftTrim(char *p)
+{
+char *q;
+q=p;
+while(*q==' ')
+{
+q++;
+}
+if(q>p) //if there were extra spaces on left, q would have been increased
+{
+while(*q!='\0')
+{
+*p=*q;
+q++;
+p++;
+}
+*p='\0';
+}
+}
+
+int main()
+{
+char a[37];
```

Comment:
is there any reason for this constraint? in all 3 programs

Reply:
no. you can take the size of string according to your requirement.

Reply:
@SidJain1412 As far as my experience goes in Codeforces and AtCoder upper limits in array constraints are usually taken as Integer.MAX_VALUE in Java and array_type[1234567] for c, c++. The program could accommodate those constraints if it's helpful.

Reply:
@priyalgupta09 I agree with @vishal-burman , please modify to remove this constraint. If the string is more than 37 chars then the program won't work right? no reason for this constraint as far as i can see

Reply:
okay i will make chnge.

Reply:
@SidJain1412  I have made changes and sent a pull request gin please see and accept my pull request